gitprompt.sh

```diff
@@ -374,6 +377,11 @@ function setGitPrompt() {
   fi
   unset OLD_GIT_PROMPT_IGNORE_SUBMODULES
 
+  if [ -z "${GIT_PROMPT_SHOW_TRACKING}" ]; then
```

Comment:
if [[ -z "${GIT_PROMPT_SHOW_TRACKING+x}" ]]; then

is used for the other variable checks, should maybe be consistent here